packages/stores/src/providers/ExchangeRatesProvider.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useSubscription } from '@deriv/api';
+import useStore from '../useStore';
+
+const ExchangeRatesProvider = ({ children }: React.PropsWithChildren<unknown>) => {
+    const { data, subscribe } = useSubscription('exchange_rates');
+    const {
+        exchange_rates: { update },
+    } = useStore();
+
+    useEffect(() => {
+        subscribe({ payload: { base_currency: 'USD' } });
```

Comment:
This will subscribe whenever the app is mounted, but we don't need this subscription on pages like dbot, cashier, or reports. 
Can we make it subscribable only if we need it?

Reply:
same :pray: